README.md

```diff
@@ -1,6 +1,9 @@
 # Zilliqa (codename _Durian_)
+
 [![Build Status](https://travis-ci.org/Zilliqa/Zilliqa.svg?branch=master)](https://travis-ci.org/Zilliqa/Zilliqa)
 
+[![codecov](https://codecov.io/gh/Zilliqa/Zilliqa/branch/master/graph/badge.svg)](https://codecov.io/gh/Zilliqa/Zilliqa)
```

Comment:
Can we align them into one line?

Reply:
Also fixed in 3548987e9ea3121c1d17abeb3ac1b281c730d75e.